README.md

```diff
@@ -1,2 +1,74 @@
-# mycalculator
-Simple calculator project made with React
+# Getting Started with Create React App
```

Comment:
**Let's make your repo look more professional by updating your `README` file**.
Checkout out the below link for help, feel free to follow this [Microverse template](https://github.com/microverseinc/readme-template) from pathwright